docs/source/customization.rst

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
There is not yet a release, should we really write changelogs for the initial release? There is a git log :wink: